### DISCUSSION TITLE
Unit of measurement of ImageRun transformation property

### DISCUSSION THREAD

**Discussion by muriIo** [dolanmiu/docx#1366] [Q&A] [answered]
Hello. I'm working with ImageRun to display some images and I would like to know what is the unit of measurement of the transformation property. 

![image](https://user-images.githubusercontent.com/61703097/148561363-d1c13d1d-f3f9-4c1c-a2f7-2a05a82d92dd.png)

Thanks!

**Comment by dolanmiu** [comment, marked as answer]
It is in TWIPS i think:

https://wordribbon.tips.net/T011388_Words_Native_Measurement_Unit.html
